packages/connectivity_plus/connectivity_plus/ios/Classes/ReachabilityConnectivityProvider.swift

```diff
@@ -4,15 +4,18 @@ import Reachability
 public class ReachabilityConnectivityProvider: NSObject, ConnectivityProvider {
   private var _reachability: Reachability?
 
-  public var currentConnectivityType: ConnectivityType {
-    let reachability = ensureReachability()
+  public var currentConnectivityTypes: [ConnectivityType] {
+    guard let reachability = _reachability else {
+      return [.none]
+    }
+    
     switch reachability.connection {
     case .wifi:
-      return .wifi
+      return [.wifi]
     case .cellular:
-      return .cellular
+      return [.cellular]
     default:
-      return .none
+      return [.none]
```

Comment:
I know this is unrelated to this PR, but, could be that the other connection types are missing in this switch statement?

Reply:
I leave the comment above the code, there are only 3 types in reachability module (wifi, cellular and none).